src/core/server/metrics/types.ts

```diff
@@ -62,3 +67,37 @@ export interface OpsMetrics {
   /** number of current concurrent connections to the server */
   concurrent_connections: OpsServerMetrics['concurrent_connections'];
 }
+
+/**
+ * an IntervalHistogram object that samples and reports the event loop delay over time.
+ * The delays will be reported in nanoseconds.
+ *
+ * @public
+ */
+export interface IntervalHistogram {
+  // The first timestamp the interval timer kicked in for collecting data points.
+  fromTimestamp: string;
+  // Last timestamp the interval timer kicked in for collecting data points.
+  lastUpdatedAt: string;
+  // The minimum recorded event loop delay.
+  min: number;
+  // The maximum recorded event loop delay.
+  max: number;
+  // The mean of the recorded event loop delays.
+  mean: number;
+  // The number of times the event loop delay exceeded the maximum 1 hour event loop delay threshold.
```

Comment:
1 hour?

Reply:
This is taken from https://nodejs.org/api/perf_hooks.html#perf_hooks_histogram_exceeds

I've opened an issue in the node repo to clarify this metric and my understanding of it: https://github.com/nodejs/node/issues/40096